### ISSUE TITLE
checksum fails for arrays with null elements

### ISSUE THREAD

**Issue by haozhun** [prestodb/presto#6450]
```
com.facebook.presto.spi.PrestoException: ARRAY comparison not supported for arrays with null elements
    at com.facebook.presto.type.TypeUtils.checkElementNotNull(TypeUtils.java:163)
    at com.facebook.presto.type.ArrayType.hash(ArrayType.java:94)
    at com.facebook.presto.type.TypeUtils.hashPosition(TypeUtils.java:60)
    at com.facebook.presto.type.MapType.hash(MapType.java:96)
    at com.facebook.presto.operator.aggregation.ChecksumAggregationFunction.input(ChecksumAggregationFunction.java:113)
    at com_facebook_presto_$gen_Map_bigint_array_varchar__Map_bigint_array_varchar__ChecksumAccumulator_29197.addInput(Unknown Source)
    at com.facebook.presto.operator.AggregationOperator$Aggregator.processPage(AggregationOperator.java:232)
    at com.facebook.presto.operator.AggregationOperator.addInput(AggregationOperator.java:155)
    at com.facebook.presto.operator.Driver.processInternal(Driver.java:384)
    at com.facebook.presto.operator.Driver.processFor(Driver.java:301)
    at com.facebook.presto.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:622)
    at com.facebook.presto.execution.TaskExecutor$PrioritizedSplitRunner.process(TaskExecutor.java:534)
    at com.facebook.presto.execution.TaskExecutor$Runner.run(TaskExecutor.java:670)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
```

**Comment by martint** [comment]
I think this was fixed recently in master

**Comment by haozhun** [comment]
We didn't change `ArrayType.hash` or `ArrayHashCodeOperator`

**Comment by martint** [comment]
Ah, it hasn't been merged, yet: https://github.com/prestodb/presto/pull/6429/files

**Comment by haozhun** [comment]
Fixed by #6429.
